config/levelbuilder.yml.erb

```diff
@@ -17,3 +17,7 @@ acapela_storage_password: !Secret
 
 # Curriculum team needs to be able to see raw HTTP error pages so they can debug the content they are implementing.
 custom_error_response: false
+
+# Configurations for making calls to the Immersive Reader API on Azure
+imm_reader_client_id:     '4fa6b522-6e6d-44c6-b023-98816b54566d'
+imm_reader_client_secret: !Secret
```

Comment:
I think these env-specific `!Secret` tags are redundant and can be removed as long as it's set from the base config. (Even if the `!Secret` tag is only stated in the base config it will still try to fetch the value from a unique, env-specific secret key.)